src/main/kotlin/me/serce/solidity/ide/run/SolidityFileType.kt

```diff
@@ -0,0 +1,37 @@
+package me.serce.solidity.ide.run
+
+import com.intellij.openapi.fileTypes.FileType
+import com.intellij.openapi.vfs.CharsetToolkit
+import com.intellij.openapi.vfs.VirtualFile
+import me.serce.solidity.ide.SolidityIcons
+import javax.swing.Icon
+
+object SolidityFileType : FileType {
```

Comment:
There is `me.serce.solidity.lang.SolidityFileType`, I think it can be reused.

Reply:
Done